assets/app/scripts/services/data.js

```diff
@@ -639,32 +646,34 @@ DataService.prototype.createStream = function(resource, name, context, opts, isR
     var callback = handle.callback;
     var objectCallback = handle.objectCallback;
     var opts = handle.opts;
+    var key = this._uniqueKeyForResourceContext(resource, objectName, context, _.get(opts, 'http.params'));
+    console.warn('TODO: .watch()/.unwatch() are NOT $http calls, so opts do not have http.params');
```

Comment:
are you just using http.params anyway? i think im ok with that since watch relies on http calls (list) and we should make a note that watch only supports a subset of the http config or something

open to alternative suggestions